src/components/FeatureViews/Scheduler/Scheduler.js

```diff
@@ -368,132 +465,141 @@ class Scheduler extends Component {
             "title": inst.name,
         }));
 
+    // go to session view
+    goToSessionView = () => (e) => {
+        // e.preventDefault();
+        const sessionID = e.event.id;
+        const courseID = e.event.extendedProps.courseID;
+        this.props.history.push(`/scheduler/view-session/${courseID}/${sessionID}`);
+    }
+
     render() {
         return (
             <Grid >
-                <Paper className="paper">
+                <Paper className="paper" style={{ padding: "2%" }}>
                     <Typography variant="h3" align="left">Scheduler</Typography>
                     <br />
-                    <div className="scheduler-header">
-                        <div className="scheduler-header-firstSet">
-                            <Grid item >
-                                <IconButton
-                                    color={this.state.calendarIcon ? "primary" : "default"}
-                                    onClick={this.changeViewToCalendar}
-                                    className={"calendar-icon"} aria-label='next-month'>
-                                    <DateRangeOutlinedIcon />
-                                </IconButton>
-                            </Grid>
-                            <Grid item >
-                                <IconButton
-                                    color={this.state.resourceIcon ? "primary" : "default"}
-                                    onClick={this.changeViewToResource}
-                                    className={"resource-icon"}
-                                    aria-label='next-month'>
-                                    <ViewListIcon />
-                                </IconButton>
-                            </Grid>
-                            <Grid item>
-                                <IconButton
-                                    className={"next-month"}
-                                    aria-label='next-month'>
-                                    <SearchIcon />
-                                </IconButton>
+                    <Grid conatiner className="scheduler-wrapper">
```

Comment:
`container` is misspelled.

Reply:
how do you catch stuff like this. mind = blown